src/lightning_habana/pytorch/strategies/single.py

```diff
@@ -124,6 +124,20 @@ def predict_step(self, *args: Any, **kwargs: Any) -> Any:
         htcore.mark_step()
         return super().predict_step(*args, **kwargs)
 
+    def on_test_end(self) -> None:
```

Comment:
parallel pllugin ?

Reply:
fp8 inference is currently supported on single card only